Create commentWriteSection

gyim1345 opened the pull request:
![commentWriteSection](https://user-images.githubusercontent.com/57941049/97868544-3c887900-1d53-11eb-953d-bb0506871641.png)

Add comment section including
- textarea for comment input
- close issue button to close issue
- comment button to submit comment on click
- on comment input, close issue button changes
- on comment input, comment button becomes functional

gyim1345 committed:
[Feat] commentWriteSection

Add comment section including
- textarea for comment input
- close issue button to close issue
- comment button to submit comment on click
- on comment input, close issue button changes
- on comment input, comment button becomes functional

gyim1345 commented:
코멘트 (생성) 기본 틀 만들었네요
코멘트 수정도 비슷한 틀인데 수정용으로도 사용할려면 

1. 수정중이다는 state를 주기 
2. 파일 하나 더 만들기

개인적으로 2번 추천... ㅋㅋ

rockpell commented on front/src/components/CommentWriteSection.js:
```diff
@@ -0,0 +1,193 @@
+        {isEmpty 
+          ? ( 
+            <>
+              <Button hoverColor="#f3f4f6" activeColor="#edeff2" onClick={closeIssue}>
```
hoverColor랑 activeColor를 태그에서 지정해줄수 있는건 처음알았네요.
수고하셨습니다!

pieisland commented on front/src/components/CommentWriteSection.js:
```diff
@@ -0,0 +1,193 @@
+  const imageURL = (userProfileURL) ? userProfileURL :  defaultUserImageUrl;
+  const inputRef = useRef();
+
+  const handleChange = (e) => {
+    if(e.target.value === ''){
+      setIsEmpty(true);
+      return
+    }
+    setIsEmpty(false);
+  }
```
어떤 역할을 하는 함수인지 이름을 보고 알 수 있으면 좋을 것 같습니다~

gyim1345 replied:
수정했습니다!

gyim1345 committed:
[Refactor] change variable name

change
handleChange -> handleTextInputChange
isEmpty -> textIsEmpty
setIsEmpty -> setTextIsEmpty

profornnan commented on front/src/components/CommentWriteSection.js:
```diff
@@ -0,0 +1,193 @@
+  content: ' ';
+    position: absolute;
+    left: -11px;
+    top: 10px;
```
styled-component에서도 :after과 :before를 사용할 수 있군요!! 불필요한 dom이 생기지 않도록 적극 활용해봐야겠네요

profornnan reviewed (approved):
고생하셨습니다!!
